deploy/iso/minikube-iso/package/podman/podman.mk

```diff
@@ -29,8 +29,6 @@ endef
 
 define PODMAN_INSTALL_TARGET_CMDS
 	$(INSTALL) -Dm755 $(@D)/bin/podman $(TARGET_DIR)/usr/bin/podman
-	$(INSTALL) -d -m 755 $(TARGET_DIR)/etc/cni/net.d/
-	$(INSTALL) -m 644 cni/87-podman-bridge.conflist $(TARGET_DIR)/etc/cni/net.d/87-podman-bridge.conflist
```

Comment:
this seems unrelated to /var and containerd

Reply:
sounds good,  I will revert that once the other PR is merged, I added it here only so I can test it with the other PR( since without it it was failing)

Reply:
done